include/awkward/kernel.h

```diff
@@ -1,14 +1,947 @@
 // BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/master/LICENSE
 
-#ifndef AWKWARD_KERNEL_H
-#define AWKWARD_KERNEL_H
+#ifndef KERNEL_H
+#define KERNEL_H
```

Comment:
The reason for `AWKWARD_KERNEL_H` is to avoid possibly conflicting names. "Kernel" is pretty generic. One of the benefits of an unusual name like "Awkward" is that it virtually guarantees uniqueness.

All of the other header-guards begin with `AWKWARD`; this one should, too.

Reply:
Sure!